### DISCUSSION TITLE
E2E CI集成

### DISCUSSION THREAD

**Discussion by BARMPlus** [Mereithhh/vanblog#248] [Ideas]
我现在使用vanblog作为我的博客，刚好看到vanblog TODO里面有`E2E CI集成`，我可以负责`E2E CI`集成这个模块，但是因为我最近工作比较忙，进度不会特别快，但是会在今年上半年完成。

**Comment by Mereithhh** [comment]
感谢！ 最近正在搞 `pnpm+ workspace`，项目结构可能会改一下，可以等我合并到 master 再动手。

**Reply by Mereithhh** [reply]
到尾声了，很快就好 :)

**Reply by BARMPlus** [reply]
好滴

**Reply by Mereithhh** [reply]
改造合并了一版，项目结构应该暂时不会有太大的变化了。
